### ISSUE TITLE
Is there a way to specify 'only' columns yet keep the serializer 'greedy'?

### ISSUE THREAD

**Issue by curtwagner1984** [n0nSmoker/SQLAlchemy-serializer#32]
I've run into an issue where I have a lot of columns on a model and I want to only exclude one specific column from serialization. Is there a way to do it besides typing in all the columns **except** the one I want to exclude in the `serialize_only` field?

**Comment by n0nSmoker** [comment]
Yes. Use `serialize_rules` with a negative rule instead of `serialize_only`, so you only have to name the field you want to drop.

## How to exclude a single column

On your model:

```python
from sqlalchemy_serializer import SerializerMixin

class MyModel(db.Model, SerializerMixin):
    __tablename__ = 'my_model'

    # exclude just this one from default serialization
    serialize_rules = ('-secret_field',)

    id = db.Column(db.Integer, primary_key=True)
    secret_field = db.Column(db.String)
    other_field = db.Column(db.String)
    # ... many more fields ...
```

Behavior:

- `serialize_rules` defines what to add/remove from the default “all SQLAlchemy fields” behavior.
- A rule like `'-secret_field'` says “include everything except `secret_field`.”
- You don’t need to list all other columns; they are included automatically.

If you only want this exclusion for a specific call, you can do it per‑call instead:

```python
result = item.to_dict(rules=('-secret_field',))
```
